node/cmd/txverifier/sui.go

```diff
@@ -18,6 +18,7 @@ import (
 
 const (
 	INITIAL_EVENT_FETCH_LIMIT = 25
+	EVENT_QUERY_INTERVAL      = 2 * time.Second
```

Comment:
nit: The Go convention for constant names is CamelCase rather than SCREAMING_SNAKE_CASE